src/commoncode/fileutils.py

```diff
@@ -62,6 +62,15 @@ def logger_debug(*args):
     def logger_debug(*args):
         return logger.debug(' '.join(isinstance(a, str) and a or repr(a) for a in args))
 
+PATH_TYPE = compat.unicode
```

Comment:
Note that compat is gone entirely and `compat.unicode` can just be replaced by `str`